ERROR - 2023-10-01 05:27:59 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'sockdb' in C:\laragon\www\jjlab3\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\laragon\www\jjlab3\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'sockDB', 3306, '', 0)
#1 C:\laragon\www\jjlab3\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\jjlab3\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\jjlab3\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\jjlab3\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\jjlab3\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\jjlab3\app\Controllers\ProductController.php(31): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\jjlab3\system\CodeIgniter.php(919): App\Controllers\ProductController->home()
#8 C:\laragon\www\jjlab3\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#9 C:\laragon\www\jjlab3\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\jjlab3\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\jjlab3\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'sockdb' in C:\laragon\www\jjlab3\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\laragon\www\jjlab3\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\jjlab3\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\jjlab3\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\jjlab3\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\jjlab3\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\jjlab3\app\Controllers\ProductController.php(31): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\jjlab3\system\CodeIgniter.php(919): App\Controllers\ProductController->home()
#7 C:\laragon\www\jjlab3\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#8 C:\laragon\www\jjlab3\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\jjlab3\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\jjlab3\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#11 {main}
CRITICAL - 2023-10-01 05:27:59 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'sockdb'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `products`', [], false)
 3 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\ProductController.php(31): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\ProductController->home()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\jjlab3\\public\\index.php')
ERROR - 2023-10-01 05:46:28 --> mysqli_sql_exception: Unknown column 'email' in 'where clause' in C:\laragon\www\jjlab3\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\jjlab3\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\laragon\www\jjlab3\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\laragon\www\jjlab3\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\laragon\www\jjlab3\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\laragon\www\jjlab3\system\Validation\Rules.php(156): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\jjlab3\system\Validation\Validation.php(309): CodeIgniter\Validation\Rules->is_unique('jjdeniega377@gm...', 'email', Array, NULL, 'email')
#6 C:\laragon\www\jjlab3\system\Validation\Validation.php(195): CodeIgniter\Validation\Validation->processRules('email', 'email', 'jjdeniega377@gm...', Array, Array)
#7 C:\laragon\www\jjlab3\system\Controller.php(150): CodeIgniter\Validation\Validation->run()
#8 C:\laragon\www\jjlab3\app\Controllers\Users.php(69): CodeIgniter\Controller->validate(Array)
#9 C:\laragon\www\jjlab3\system\CodeIgniter.php(919): App\Controllers\Users->register()
#10 C:\laragon\www\jjlab3\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#11 C:\laragon\www\jjlab3\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\jjlab3\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\jjlab3\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#14 {main}
CRITICAL - 2023-10-01 05:46:28 --> Unknown column 'email' in 'where clause'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\Rules.php(156): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(309): CodeIgniter\Validation\Rules->is_unique('jjdeniega377@gmail.com', 'email', [...], null, 'email')
 4 SYSTEMPATH\Validation\Validation.php(195): CodeIgniter\Validation\Validation->processRules('email', 'email', 'jjdeniega377@gmail.com', [...], [...])
 5 SYSTEMPATH\Controller.php(150): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\Users.php(69): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Users->register()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\jjlab3\\public\\index.php')
ERROR - 2023-10-01 05:51:21 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in C:\laragon\www\jjlab3\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\jjlab3\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\laragon\www\jjlab3\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\jjlab3\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\jjlab3\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\jjlab3\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\jjlab3\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\jjlab3\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\jjlab3\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\laragon\www\jjlab3\app\Controllers\Users.php(79): CodeIgniter\BaseModel->save(Array)
#9 C:\laragon\www\jjlab3\system\CodeIgniter.php(919): App\Controllers\Users->register()
#10 C:\laragon\www\jjlab3\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#11 C:\laragon\www\jjlab3\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\jjlab3\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\jjlab3\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#14 {main}
CRITICAL - 2023-10-01 05:51:21 --> Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`name`, `email`, `password`, `created_at`) VALUES (:name:, :email:, :password:, :created_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Users.php(79): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Users->register()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\jjlab3\\public\\index.php')
ERROR - 2023-10-01 05:52:51 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in C:\laragon\www\jjlab3\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\jjlab3\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\laragon\www\jjlab3\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\jjlab3\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\jjlab3\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\jjlab3\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\jjlab3\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\jjlab3\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\jjlab3\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\laragon\www\jjlab3\app\Controllers\Users.php(79): CodeIgniter\BaseModel->save(Array)
#9 C:\laragon\www\jjlab3\system\CodeIgniter.php(919): App\Controllers\Users->register()
#10 C:\laragon\www\jjlab3\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#11 C:\laragon\www\jjlab3\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\jjlab3\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\jjlab3\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#14 {main}
CRITICAL - 2023-10-01 05:52:51 --> Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`name`, `email`, `password`, `created_at`) VALUES (:name:, :email:, :password:, :created_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Users.php(79): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Users->register()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\jjlab3\\public\\index.php')
ERROR - 2023-10-01 05:59:21 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in C:\laragon\www\jjlab3\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\jjlab3\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\laragon\www\jjlab3\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\jjlab3\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\jjlab3\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\jjlab3\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\jjlab3\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\jjlab3\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\jjlab3\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\laragon\www\jjlab3\app\Controllers\Users.php(79): CodeIgniter\BaseModel->save(Array)
#9 C:\laragon\www\jjlab3\system\CodeIgniter.php(919): App\Controllers\Users->register()
#10 C:\laragon\www\jjlab3\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#11 C:\laragon\www\jjlab3\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\jjlab3\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\jjlab3\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#14 {main}
CRITICAL - 2023-10-01 05:59:21 --> Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`name`, `email`, `password`, `created_at`) VALUES (:name:, :email:, :password:, :created_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Users.php(79): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Users->register()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\jjlab3\\public\\index.php')
ERROR - 2023-10-01 06:10:03 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in C:\laragon\www\jjlab3\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\jjlab3\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\laragon\www\jjlab3\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\jjlab3\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\jjlab3\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\jjlab3\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\jjlab3\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\jjlab3\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\jjlab3\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\laragon\www\jjlab3\app\Controllers\Users.php(79): CodeIgniter\BaseModel->save(Array)
#9 C:\laragon\www\jjlab3\system\CodeIgniter.php(919): App\Controllers\Users->register()
#10 C:\laragon\www\jjlab3\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#11 C:\laragon\www\jjlab3\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\jjlab3\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\jjlab3\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#14 {main}
CRITICAL - 2023-10-01 06:10:03 --> Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`name`, `email`, `password`, `created_at`) VALUES (:name:, :email:, :password:, :created_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Users.php(79): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Users->register()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\jjlab3\\public\\index.php')
ERROR - 2023-10-01 06:12:21 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in C:\laragon\www\jjlab3\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\jjlab3\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\laragon\www\jjlab3\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\jjlab3\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\jjlab3\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\jjlab3\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\jjlab3\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\jjlab3\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\jjlab3\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\laragon\www\jjlab3\app\Controllers\Users.php(79): CodeIgniter\BaseModel->save(Array)
#9 C:\laragon\www\jjlab3\system\CodeIgniter.php(919): App\Controllers\Users->register()
#10 C:\laragon\www\jjlab3\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#11 C:\laragon\www\jjlab3\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\jjlab3\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\jjlab3\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#14 {main}
CRITICAL - 2023-10-01 06:12:21 --> Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`name`, `email`, `password`, `created_at`) VALUES (:name:, :email:, :password:, :created_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Users.php(79): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Users->register()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\jjlab3\\public\\index.php')
ERROR - 2023-10-01 06:12:26 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in C:\laragon\www\jjlab3\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\jjlab3\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\laragon\www\jjlab3\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\jjlab3\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\jjlab3\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\jjlab3\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\jjlab3\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\jjlab3\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\jjlab3\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\laragon\www\jjlab3\app\Controllers\Users.php(79): CodeIgniter\BaseModel->save(Array)
#9 C:\laragon\www\jjlab3\system\CodeIgniter.php(919): App\Controllers\Users->register()
#10 C:\laragon\www\jjlab3\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#11 C:\laragon\www\jjlab3\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\jjlab3\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\jjlab3\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#14 {main}
CRITICAL - 2023-10-01 06:12:26 --> Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`name`, `email`, `password`, `created_at`) VALUES (:name:, :email:, :password:, :created_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Users.php(79): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Users->register()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\jjlab3\\public\\index.php')
CRITICAL - 2023-10-01 15:26:40 --> Invalid file: "jhome.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('jhome.php')
 2 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('jhome', [], true)
 3 APPPATH\Controllers\ProductController.php(33): view('jhome', [...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\ProductController->home()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\jjlab3\\public\\index.php')
CRITICAL - 2023-10-01 15:27:26 --> Invalid file: "body.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('body.php')
 2 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('body', [], true)
 3 APPPATH\Views\plate.php(4): view('body')
 4 SYSTEMPATH\View\View.php(226): include('C:\\laragon\\www\\jjlab3\\app\\Views\\plate.php')
 5 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('plate', [], true)
 7 APPPATH\Controllers\ProductController.php(33): view('plate', [...])
 8 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\ProductController->home()
 9 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
10 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\jjlab3\\public\\index.php')
CRITICAL - 2023-10-01 15:27:41 --> Invalid file: "home.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('home.php')
 2 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('home', [], true)
 3 APPPATH\Controllers\ProductController.php(33): view('home', [...])
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\ProductController->home()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\jjlab3\\public\\index.php')
CRITICAL - 2023-10-01 15:28:11 --> Invalid file: "body.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('body.php')
 2 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('body', [], true)
 3 APPPATH\Views\home.php(4): view('body')
 4 SYSTEMPATH\View\View.php(226): include('C:\\laragon\\www\\jjlab3\\app\\Views\\home.php')
 5 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('home', [], true)
 7 APPPATH\Controllers\ProductController.php(33): view('home', [...])
 8 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\ProductController->home()
 9 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
10 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\jjlab3\\public\\index.php')
CRITICAL - 2023-10-01 15:28:37 --> Invalid file: "inc/header.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('inc/header.php')
 2 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('inc/header', [], true)
 3 APPPATH\Views\home.php(5): view('inc/header')
 4 SYSTEMPATH\View\View.php(226): include('C:\\laragon\\www\\jjlab3\\app\\Views\\home.php')
 5 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('home', [], true)
 7 APPPATH\Controllers\ProductController.php(33): view('home', [...])
 8 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\ProductController->home()
 9 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
10 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\jjlab3\\public\\index.php')
CRITICAL - 2023-10-01 15:29:50 --> Invalid file: "inc/header.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('inc/header.php')
 2 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('inc/header', [], true)
 3 APPPATH\Views\home.php(5): view('inc/header')
 4 SYSTEMPATH\View\View.php(226): include('C:\\laragon\\www\\jjlab3\\app\\Views\\home.php')
 5 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('home', [], true)
 7 APPPATH\Controllers\ProductController.php(33): view('home', [...])
 8 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\ProductController->home()
 9 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
10 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\jjlab3\\public\\index.php')
